exercises/complex-numbers/complex_numbers_test.py

```diff
@@ -4,74 +4,64 @@
 
 from complex_numbers import ComplexNumber
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

Comment:
Can you move the comment down by a line for the sake of consistency? (ie. two blank rows before, one after)